br/pkg/lightning/restore/table_restore.go

```diff
@@ -314,11 +314,20 @@ func (tr *TableRestore) restoreEngines(pCtx context.Context, rc *Controller, cp
 						dataWorker := rc.closedEngineLimit.Apply()
 						defer rc.closedEngineLimit.Recycle(dataWorker)
 						err = tr.importEngine(ctx, dataClosedEngine, rc, eid, ecp)
+						if rc.status != nil {
+							for _, chunk := range ecp.Chunks {
+								rc.status.FinishedFileSize.Add(chunk.FileMeta.FileSize)
```

Comment:
Multiple chunks may share a same file when we split large csv files. The correct value should be `chunk.Chunk.EndOffset - chunk.Chunk.Offset`.

Reply:
But if it started since a checkpoint, we shall calculate the total size rather than how much it does in this job.